src/main/java/org/sagebionetworks/bridge/workerPlatform/bridge/BridgeHelper.java

```diff
@@ -265,6 +267,30 @@ public Upload getUploadByRecordId(String recordId) throws IOException {
         return clientManager.getClient(ForWorkersApi.class).getUploadByRecordId(recordId).execute().body();
     }
 
+    /** Get account summaries by caller's appId and org */
+    public List<AccountSummary> getAccountSummariesForApp(String appId, String orgId, int offsetBy, int pageSize,
+                                                          String studyId) throws IOException {
+        AccountSummarySearch search = new AccountSummarySearch().offsetBy(offsetBy);
+
+        if (studyId != null) {
+            search.enrolledInStudyId(studyId);
+        } else if (orgId != null) {
+            search.orgMembership(orgId);
+        }
+
+        if (pageSize > 0) {
+            search.pageSize(PARTICIPANT_PAGE_SIZE);
+        }
```

Comment:
Bug: We set the file size to the default page size instead of the passed in page size.